lib/ts/recipe/emailpassword/api/passwordReset.ts

```diff
@@ -39,11 +41,11 @@ export default async function passwordReset(
         value: string;
     }[] = await validateFormFieldsOrThrowError(
         options.config.resetPasswordUsingTokenFeature.formFieldsForPasswordResetForm,
-        (await options.req.getJSONBody()).formFields,
+        requestBody.formFields,
         tenantId
     );
 
-    let token = (await options.req.getJSONBody()).token;
+    let token = requestBody.token;
```

Comment:
you should keep the old code here where we are reading it twice, and add a test for the froamework (at least for express and nextjs) where we cal this API and make sure it doesn't fail. You should check the test by removing the caching mechanism and manually making sure that the test does fail, and then readd the caching logic to make sure the test then passes.

Reply:
Or if you don't like that, then have some other way to add tests for this PR.